synapse/lib/storm.py

```diff
@@ -921,6 +971,15 @@ def _stormOperLift(self, query, oper):
 
         [query.add(tufo) for tufo in self.stormTufosBy(by, prop, valu, limit=limit)]
 
+    def _stormOperLimit(self, query, oper):
+        args = oper[1].get('args',())
+        if len(args) != 1:
+            raise s_common.BadSyntaxError(mesg='limit(<size>)')
```

Comment:
we should have an example which throws BadSyntaxError since this is currently untested.